### ISSUE TITLE
The Last Hero of Nostalgaia black background in game (Android)

### ISSUE THREAD

**Issue by brujo5** [yuzu-emu/yuzu#10857]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Acces e2967c2a65

### Description of Issue

The Last Hero of Nostalgaia black background in game 

![Screenshot_20230619-194757_yuzu](https://github.com/yuzu-emu/yuzu/assets/4172898/1650f941-5e0a-41d4-92a8-6acdaddff8ee)

### Expected Behavior

it should look like this

![Screenshot_20230619-200052_Egg_NS_Emulador](https://github.com/yuzu-emu/yuzu/assets/4172898/d1301340-a994-4956-9e62-17010b5c5833)

### Reproduction Steps

play normally

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11795482/yuzu_log.txt)

### System Configuration

CPU: sd860
GPU/Driver:  adreno 640/turnip driver VK 1.3.251 
RAM: 8gb
OS: android 13

**Comment by Squall-Leonhart** [comment]
adreno 640 and lower have graphical issues on the official turnip driver, need to use the skyline modded one.

**Comment by Squall-Leonhart** [comment]
**It is though**

**Comment by Squall-Leonhart** [comment]
It's almost like **Not every emulator has the same requirements**

**Comment by brujo5** [comment]
You are right but turnip works perfectly in the other emulators called skyline or eggy and we all know that the shady emulator (eggy) uses the yuzu code and does not present any rendering problem that I have mentioned with any game here and even the drivers from qualcomm are usable and have no rendering issues.
the only difference with the turnip drivers shared by pixelon (skyline dev) was the new kgsl rewrite (which fixed the emulator crash after 10-15min) but this fix is already in the sorce code from months ago, so now no difference now
